docs/4.tutorials-and-examples/6.custom-webchat.mdx

```diff
@@ -13,51 +13,86 @@ This template shows how to create custom elements for webchat. All the custom co
 **src/webchat/index.js**
 
 ```javascript
-import Logo from '../assets/bot_vader.jpg'
-import { MyReply } from './myReply'
-
-export const theme = {
-  brandColor: '#ff0000',
-  textPlaceholder: 'Please, type your message:',
-  title: 'Custom Webchat',
-  brandIconUrl: Logo,
-  customReply: MyReply
+export const webchat = {
+  theme: {},
+
+  persistentMenu: [],
+
+  blockInputs: [],
+
+  // Webchat listeners
+  onInit: app => {},
+  onOpen: app => {},
+  onClose: app => {},
+  onMessage: app => {}
 }
 ```
 
-This is the main file of the webchat config. In this file, we export all the options that are going to be changed of the default webchat config.
+This is just a boilerplate code containing all the allowed definitions for your Webchat.
+- **theme**: In this object you can define the styles for your Webchat components.
+- **persistentMenu**: This will enable a persistent menu, which is a component that will help users to access the core functionalities of the bot at any point in the conversation.
+- **blockInputs**: Sometimes you would want to forbide some user inputs of the user. This permits you to define which are the inputs not allowed.
+- **Listeners**: Listeners are useful in order to react to some events when using the webchat.  
+
+Let's see how we can use these objects:
+
+## theme
 
-This are the available options that Botonic allows you to modify:
+These are the available styles that Botonic allows you to modify:
+### General Styling
+- **style:** the main characteristics of webchat such as its size, its background color...
 
-- **brandColor:** the main color of the webchat widget.
+* **brandColor:** the main color palette of the webchat widget.
 
-* **textPlaceholder:** text placeholder in the input text.
+- **textPlaceHolder:** text placeholder in the input text.
 
-- **title:** the title it's dsiplayed in the header of the widget.
+* **botLogoChat:** the icon representing the bot in the chat.
 
-* **brandIconUrl:** it can be a http url, or an image. It's the image displayed in the header.
+- **brandIconUrl:** launcher icon for toggling the webchat.
 
-- **customReply:** React Component that describes how the quickreplies are shown.
+* **customTriggerButton:** React component that is injected in the Launcher icon.
 
-* **customButton:** React Component that describes how the button is displayed.
+### Header Styling
+- **headerTitle:** display title of the webchat.
+
+* **headerSubtitle:** display subtitle of the webchat.
+
+- **headerImage:** the main color of the webchat widget.
```

Comment:
"main color" -> background image?

Reply:
- **headerImage:** the main logo appearing in the header.